tensorflow/core/kernels/mkl/mkl_fused_batch_norm_op_test.cc

```diff
@@ -118,10 +124,94 @@ class CommonTestUtilities : public OpsTestBase {
     test::ExpectClose(batch_var, mkl_batch_var, 1e-5);
   }
 
+  static void VerifyTensorsCloseForGrad(const float epsilon,
+                                        const GraphRunnerGrad& run,
+                                        const GraphRunnerGrad& run_mkl) {
+    int batch = 2;
+    int height = 8;
+    int width = 8;
+    int depth = 1;
+    DataType dtype = DataTypeToEnum<T>::v();
+
+    Tensor input(dtype, {batch, height, width, depth});
+    input.flat<T>() = input.flat<T>().template setRandom<random_gen_>();
+    Tensor filter(dtype, {3, 3, 1, 6});
```

Comment:
Would you mind defining variables for the sizes instead? For example,
```c++
Tensor filter(dtype, {filter_height, filter_width, in_channels, out_channels});
...
Tensor y_backprop(dtype, {batch, height, width, out_channels});
...
Tensor scale(type, {out_channels});
...
```

Reply:
Updated per suggestion in the new commit. Please check.